### PR TITLE
Fix - Show cross in IE

### PR DESCRIPTION
**Pull request by NaomiFPassarelli**
In Internet Explorer the cross on the right side was not shown.
So I encode svg.

look at this: https://codepen.io/tigt/post/optimizing-svgs-in-data-uris

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,4 @@ demo
 dist
 coverage
 .DS_Store
-.npmignore
\ No newline at end of file
+.npmignore
```
Comment (dennybiasiolli) [inline .gitignore]: This line isn't necessary ;)

File: src/alert/Close.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const Close = glamorous('div')({
     opacity: '0.5'
   }
 }, props => ({
-  background: `url('data:image/svg+xml;utf8,<svg fill="${props.glam.dark ? '%23fff' : '%23333'}" xmlns="http://www.w3.org/2000/svg" height="48" viewBox="0 0 48 48" width="48"><path d="M38 12.83l-2.83-2.83-11.17 11.17-11.17-11.17-2.83 2.83 11.17 11.17-11.17 11.17 2.83 2.83 11.17-11.17 11.17 11.17 2.83-2.83-11.17-11.17z"/><path d="M0 0h48v48h-48z" fill="none"/></svg>') no-repeat`,
+  background: `url("data:image/svg+xml;charset=utf8,%3Csvg%20fill%3D%22%24%7Bprops.glam.dark%20%3F%20%27%2523fff%27%20%3A%20%27%2523333%27%7D%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20height%3D%2248%22%20viewBox%3D%220%200%2048%2048%22%20width%3D%2248%22%3E%3Cpath%20d%3D%22M38%2012.83l-2.83-2.83-11.17%2011.17-11.17-11.17-2.83%202.83%2011.17%2011.17-11.17%2011.17%202.83%202.83%2011.17-11.17%2011.17%2011.17%202.83-2.83-11.17-11.17z%22%2F%3E%3Cpath%20d%3D%22M0%200h48v48h-48z%22%20fill%3D%22none%22%2F%3E%3C%2Fsvg%3E") no-repeat`,
```
Comment (dennybiasiolli) [inline src/alert/Close.js]: Encoding SVG you have removed `${props.glam.dark ? '%23fff' : '%23333'}` condition

File: yarn.lock [hunk 3]
```diff
@@ -856,6 +856,10 @@ babel-types@^6.18.0, babel-types@^6.19.0, babel-types@^6.23.0, babel-types@^6.24
     lodash "^4.2.0"
     to-fast-properties "^1.0.1"
 
+babel@^6.23.0:
```
Comment (dennybiasiolli) [inline yarn.lock]: Committing `yarn.lock` is not necessary in this PR 😃

**Comment by dennybiasiolli** [comment]
Hi @NaomiFPassarelli, thanks for changes, but tests are broken, it seems that `src/alert/Close.js` contains some wrong code.
I suggest you to squash all your commits into a single one, changing only the file you need. Other than that you can test your code with `npm run test` before submitting the PR, if tests passes you can proceed with the PR 😉

**Comment by schiehll** [comment]
Hey @NaomiFPassarelli, thank you very much for the PR and sorry about not being able to help you to get it merged.

v3 should fix this issue and v2 will not be maintained anymore, so I will be closing this PR, hope it's ok.

**Pull request closed without merging**